packages/messaging-system/.eslintrc.json

```
@@ -0,0 +1,22 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
```

Can you add jest here?
```suggestion
        "es2021": true,
        "jest": true
```